examples/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "examples"
+version = "0.0.0"
+authors = ["The ERDOS Team"]
+edition = "2018"
+publish = false
+
+[dependencies]
+erdos = { version = "0.3.1", path = "../erdos", features = ["ros"] }
+
+abomonation = "0.7.3"
+abomonation_derive = "0.5.0"
+async-trait = "0.1.18"
+bincode = "1.3.1"
+bytes = "1.0.0"
+byteorder = "1.3.4"
+clap = "2.33.0"
+futures = "0.3.5"
+futures-delay-queue = "0.4.2"
+futures-intrusive = "0.4"  # Required for futures-delay-queue
+futures-util = "0.3.5"
+petgraph = "0.5.0"
+pyo3 = { version = "0.14.3", optional = true }
+rand = "0.3"
+rosrust = { version = "0.9", optional = true }
+rosrust_msg = { version = "0.1", optional = true }
+serde = { version = "1.0.115", features = ["derive"] }
+tracing = "0.1.29"
+tracing-appender = "0.2.0"
+tracing-subscriber = "0.3.1"
+tokio = { version = "1.8.1", features = ["full"] }
+tokio-util = { version = "0.6.7", features = ["codec"] }
+tokio-serde-bincode = "0.2.1"
+uuid = { version = "0.7", features = ["v4", "v5", "serde"] }
```

```suggestion
rosrust_msg = { version = "0.1", optional = true }
tracing = "0.1.29"
```

Don't need most of these dependencies.